src/Services/WebfontCSSGenerator/WebfontCSSGenerator.php

```diff
@@ -204,6 +207,7 @@ protected function makeFontStyleCSS(string $familyName, string $styleName): stri
             . "\tfont-style: ".($style->isItalic ? 'italic' : 'normal').";\n"
             . (isset($files['eot']) ? "\tsrc: url(".CSSHelpers::formatString($files['eot']).");\n" : '')
             . "\tsrc: ".implode(', ', $sources).";\n"
+            . (isset($fontDisplay) && $fontDisplay !== '' ? "\tfont-display: " . $fontDisplay . ";\n" : "")
```

Comment:
Considering the previous notice, the value must be screened before inserting into the CSS code.

Also `isset($fontDisplay) && $fontDisplay !== ''` may be shortened to `($fontDisplay ?? '') !== ''`.